src/storage/data_table.cpp

```diff
@@ -425,7 +343,7 @@ bool DataTable::CompareAndSwapVersionPtr(const TupleSlot slot, const TupleAccess
 }
 
 RawBlock *DataTable::NewBlock() {
-  RawBlock *new_block = block_store_->Get();
+  RawBlock *new_block = block_store_.operator->()->Get();
```

Comment:
Why did you make this change?  If the previous code was no longer working, then we have a problem with the block store code and should fix that rather than make this change.

Reply:
done